oc-micro-api/public/javascripts/index.js

```diff
@@ -0,0 +1,13 @@
+((params, $) => {
+  setTimeout(() => {
+    $.ajax({
+      type: 'GET',
+      url: `${params.baseUrl}oc-micro-api/?render=api`,
+      beforeSend: (xhr) => { xhr.setRequestHeader('Accept', 'application/vnd.oc.unrendered+json'); },
```

Comment:
hi @ gregoryStarr this is how you would call an OpenComponent w/ AJAX and consuming only the data part